src/System.Xml.XmlSerializer/tests/Performance/XsPerformanceTest.cs

```diff
@@ -17,7 +17,12 @@ public class XsPerformanceTest
     {
         public static IEnumerable<object[]> SerializeMemberData()
```

Comment:
Nit: It's a bit odd to convert config to object array and return IEnumerable<object[]> as we now have PerfTestConfig type.